src/import-data/ImportDataDestination.ts

```diff
@@ -219,6 +220,13 @@ export const ImportDataDestination = (props: ImportDataDestinationProps): ReactN
           ]),
       ]),
       importMayTakeTime && div({ style: { marginTop: '0.5rem', lineHeight: '1.5' } }, [importMayTakeTimeMessage]),
+      !!selectedWorkspace &&
+        h(Fragment, [
+          h(WorkspacePolicies, { workspace: selectedWorkspace }),
+          !!isProtectedData &&
+            isAzureWorkspace(selectedWorkspace) &&
+            div([p(['Importing this data may add:']), ul([li(['Additional access controls'])])]),
```

Comment:
I did reduce the vertical spacing a bit in `WorkspacePolicies`, so it is now not consistent with the default (and with this "Additional access controls" bit.

Does this "Additional access controls" _need_ to be a list, given that it will never be more than one thing? Can you just add "Importing this data may add additional access controls"?

Reply:
This is a good question!  Do you mind bubbling this up in https://www.figma.com/file/3u8nPWUuihYARDb2DhMI97/journey?type=design&node-id=0-1&mode=design&t=lQH3tD8hrYonYU3r-0 so it gets :eyes: from both Kate & Beth?

I'll plan to stick with what's implemented here for the sake of getting this landed but happy to iterate as needed.

Reply:
@jladieu I will let you update the design documentation. Note that if you keep the UI as implemented, you should adjust the style of the ul/li block so that it is consistent with the vertical styling in `WorkspacePolicies`.

        { style: { marginBlockStart: '0.5rem', marginBlockEnd: '0.5rem' } },